rosbag2/include/rosbag2/storage_options.hpp

```diff
@@ -25,6 +25,7 @@ struct StorageOptions
 public:
   std::string uri;
   std::string storage_id;
+  uint64_t max_bagfile_size;
```

Comment:
do you mind having this field enhanced with a little comment that the size is to be provided in bytes?

Reply:
Suffixing variables with units is also good. (max_bagfile_size_in_bytes)

Reply:
I'm making `should_split_bagfile` private. I'll also add a comment to `max_bagfile_size` to specify that its in bytes.